tools/postgres/schema.sql

```diff
@@ -44,10 +44,10 @@ CREATE TABLE webknossos.annotations(
   tags TEXT[] NOT NULL DEFAULT '{}',
   tracingTime BIGINT,
   typ webknossos.ANNOTATION_TYPE NOT NULL,
-  othersMayEdit BOOLEAN NOT NULL DEFAULT false,
+  othersMayEdit BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
@fm3 During development, I noticed that the case spelling off false is not consistent and I corrected here. Do you think it is ok to keep this?